mizar/dp/mizar/workflows/builtins/networkpolicies/create.py

```diff
@@ -76,12 +76,22 @@ def handle_networkpolicy_create_update(self, name, pod_label_dict, policy_types)
 
     def generate_data_for_networkpolicy_ingress(self, ep):
         data = self.init_data_for_networkpolicy()
-        #TODO build ingress data
+        gress_type = "ingress"
+
+        for networkpolicy_name in ep.ingress_networkpolicies:
+            networkpolicy = networkpolicy_opr.get_networkpolicy(networkpolicy_name)
+            self.fill_data_from_onegress(data, gress_type, networkpolicy)
+        self.build_gressdata(data, ep)
```

Comment:
same with the naming. gress is not good naming. can you name it something close to industry standard so that other developer can understand better?

Reply:
Fixed